src/query/src/sql.rs

```diff
@@ -224,6 +227,18 @@ fn describe_column_semantic_types(
     ))
 }
 
+pub fn stmt_options_to_table_options(opts: &[SqlOption]) -> table::error::Result<TableOptions> {
```

Comment:
question: Should we place this in the sql crate instead of the query crate?

Reply:
> question: Should we place this in the sql crate instead of the query crate?

Good question. Another approach considers changing` options: Vec<SqlOption>,` to `options: Vec<String, String>` directly (like we did in `CopyFrom`), then we won't touch any crate except the `sql`. 
BTW, use if we try to put this function in the `sql` crate will result in importing the `table`, which I'm afraid will incur cycle import.

Reply:
Maybe we could implement a method for `CreateTable`, converts `options: Vec<SqlOption>` to `options: Vec<String, String>`;